core/packages/test/scripts/deploy-polkadot.sh

```diff
@@ -11,7 +11,7 @@ generate_chain_spec() {
     while [ -z "$initial_beacon_block" ] || [ "$initial_beacon_block" == "0x0000000000000000000000000000000000000000000000000000000000000000" ]
     do
         echo "Waiting for beacon chain to finalize to get initial block..."
-        initial_beacon_block=$(curl -s "$beacon_endpoint_http/eth/v1/beacon/states/head/finality_checkpoints" \
+        initial_beacon_block=$(curl -s "$beacon_endpoint_http/eth/v1/beacon/states/justified/finality_checkpoints" \
```

Comment:
@claravanstaden not sure this is nessessary, but with `head` sometimes failed in my setup

Reply:
Hmmm, I wonder if we shouldn't rather retry getting the head finalized checkpoint and bootstrap if it fails?

Reply:
I would assume both as check point will be fine, but anyway we can still use `head` and retry util finished so I'll just reverted.

Reply:
`justified` checkpoints can still be reorged (as I understand it) and so `finalized` should be used instead for safety.